.github/workflows/stale.yaml

```diff
@@ -12,15 +13,16 @@ jobs:
     if: github.repository == 'aws/karpenter'
     name: Stale issue bot
     steps:
-      - uses: actions/stale@v5.0.0
+      - uses: actions/stale@v8.0.0
         with:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
-          stale-issue-message: 'Labeled for closure due to inactivity in 10 days.'
-          exempt-issue-labels: 'bug,chore,roadmap,help-wanted,burning,feature,documentation,operational-excellence,good-first-issue'
+          stale-issue-message: 'This issue has been inactive for 14 days. StaleBot will close this stale issue after 14 more days of inactivity.'
+          exempt-issue-labels: 'bug,chore,feature,documentation,testing,operational-excellence'
           stale-issue-label: 'lifecycle/stale'
           close-issue-label: 'lifecycle/closed'
-          remove-stale-when-updated: true
-          days-before-stale: 20
-          days-before-close: 10
-          operations-per-run: 300
-          only-pr-labels: 'close-pr-if-stale' # Because we don't want the bot to close PRs for now.
+          stale-pr-message: 'This PR has been inactive for 14 days. StaleBot will close this stale PR after 14 more days of inactivity.'
+          stale-pr-label: 'lifecycle/stale'
+          close-pr-label: 'lifecycle/closed'
```

Comment:
can we add exempt-pr-labels? Maybe for `blocked` PRs?

Reply:
That's a good callout. I wanted to reduce the label set for this but I think that `blocked` is probably a good one